src/utils.js

```diff
@@ -25,13 +26,23 @@ export const parseInclude = request => {
   const models = getModels(request);
   if (models.isBoom) return models;
 
-  return include.map(a => {
-    const singluarOrPluralMatch = Object.keys(models).find((modelName) => {
-      const { _singular, _plural } = models[modelName];
-      return _singular === a || _plural === a;
-    });
+  return include.map(b => {
+    let a = b;
+    try {
+      if (joi.string().regex(/^\{.*?"model":.*?\}$/)) {
```

Comment:
I get it. I was opting for: "if you pass JSON, we're going to skip validating it … it's too complex to do correctly". 

Here's the problem I see with:

```js
joi.object().keys({
        model: joi.string().valid(...modelAssociations),
        where: joi.object().keys({
          ...attributeValidation,
          ...sequelizeOperators,
        }),
      })
```

It ensures you get a valid JS object to validate. But you won't. You'll get a string that contains JSON. That's why I was opting for the regex of a string. Did I get confused there?